controller_manager:
  ros__parameters:
    update_rate: 100

    minimal_example_controller:
      type: minimal_example_controller/MinimalExampleController

minimal_example_controller:
  ros__parameters:
    joints:
      - joint1
      - joint2
    interface_name: effort